proto/proto/store/rpc.proto

```diff
@@ -309,18 +309,20 @@ message AccountVaultUpdate {
 // Specifies note tags that client is interested in. The server will return the first block which
 // contains a note matching `note_tags` or the chain tip.
 message SyncNotesRequest {
-    // Last block known by the client. The response will contain data starting from the next block,
-    // until the first block which contains a note of matching the requested tag.
-    fixed32 block_num = 1;
+    // Block range from which to start synchronizing.
+    //
+    // If the `block_to` is specified, this block must be close to the chain tip (i.e., within 30 blocks),
+    // otherwise an error will be returned.
```

Comment:
This part is not correct, right? `block_to` could be anything for note syncing.

Reply:
Yes, does not applies here.